src/cmd/manage_nsfs.js

```diff
@@ -338,6 +338,7 @@ async function fetch_account_data(action, user_input) {
             distinguished_name: user_input.user,
             uid: user_input.user ? undefined : user_input.uid,
             gid: user_input.user ? undefined : user_input.gid,
+            supplemental_groups: _.isUndefined(user_input.supplemental_groups) ? undefined : String(user_input.supplemental_groups),
```

Comment:
looks like supplemental_groups loading is missing in bucketspace_fs.read_account_by_access_key()